src/types/types.go

```diff
@@ -60,6 +66,7 @@ type VmData struct {
 	TemplateName string
 	GroupUUID    string `json:"group_id"`
 	NodeUUID     string `json:"node_id"`
+	Arch         string `json:"arch"`
```

Comment:
Is the arch field always present? According to the [docs](https://docs.veertu.com/anka/anka-build-cloud/working-with-controller-and-api/#vm-instance):
> All fields but the following are omitted if empty: vmid, group_id, instance_state, anka_registry, ts, cr_time, progress, vlan and usb_device.